internal/pkg/describe/env.go

```diff
@@ -131,6 +133,32 @@ func (d *EnvDescriber) Describe() (*EnvDescription, error) {
 	return d.description, nil
 }
 
+// Manifest returns the contents of the manifest used to deploy an environment stack.
+func (d *EnvDescriber) Manifest() ([]byte, error) {
+	tpl, err := d.cfn.StackMetadata()
+	if err != nil {
+		return nil, err
```

Comment:
We might want to wrap this error 🤔

Reply:
I didn't on purpose because the error msg from the downstream package already has all the info 👍